master/docs/manual/cfg-global.rst

```diff
@@ -553,6 +553,127 @@ This value can also be changed via a reconfig.
 
 Read more about metrics in the :ref:`Metrics` section in the developer documentation.
 
+.. bb:cfg:: stats-service
+
+Statistics Service
+~~~~~~~~~~~~~~~~~~
+
+The Statistics Service (or stats service) is a new service being introduced in Buildbot Nine.
+This service supports for collecting arbitrary data from within a running Buildbot instance and export it do a number of storage backends.
+Currently, only `InfluxDB <http://influxdb.com>`_ is supported as a storage backend.
+Also, InfluxDB (or any other storage backend) is not a mandatory dependency.
+Buildbot can run without it although :class:`StatsService` will be of no use in such a case.
+At present, :class:`StatsService` can only keep track of build properties and build times (start, end, duration).
+
+Example usage:
+
+::
+
+    captures = [stats.CaptureProperty('runtests', 'tree-size-KiB'),
+                stats.CaptureBuildDuration('runtests')]
+    c['services'] = []
+    c['services'].append(stats.StatsService(
+        storage_backends=[
+            stats.InfluxStorageService('localhost', 8086, 'root', 'root', 'test', captures)
+        ], name="StatsService"))
```

Comment:
Thanks !

A small note there, while it's pretty clear that 'tree-size-KiB' is a property, it might not be obvious that 'runtests' is the name of the builder, choosing a more explicit name here might help.